includes/widgets/class-wc-widget-layered-nav.php

```diff
@@ -219,9 +219,11 @@ protected function layered_nav_dropdown( $terms, $taxonomy, $query_type ) {
 			$term_counts          = $this->get_filtered_term_product_counts( wp_list_pluck( $terms, 'term_id' ), $taxonomy, $query_type );
 			$_chosen_attributes   = WC_Query::get_layered_nav_chosen_attributes();
 			$taxonomy_filter_name = str_replace( 'pa_', '', $taxonomy );
+			$taxonomy_label       =  wc_attribute_label( $taxonomy );
+			$any_label            = apply_filters( 'woocommerce_layered_nav_any_label', sprintf( __( 'Any %s', 'woocommerce' ), $taxonomy_label ), $taxonomy_label );
```

Comment:
Please include `$taxonomy` in the filter, since this should help things more easy if you can test for the taxonomy slug/name and not only by the label.